toolkit/cli/node-commands/README.md

```diff
@@ -0,0 +1,219 @@
+
+**Usage:**
+```bash
+partner-chains-node sign-block-producer-metadata [OPTIONS]
+```
+
+#### 7. Smart Contracts (`smart-contracts`)
+Provides a comprehensive suite of commands for interacting with Partner Chain smart contracts deployed on Cardano.
+
+**Usage:**
+```bash
+partner-chains-node smart-contracts [SUBCOMMAND]
+```
+
+#### 8. Setup Wizards (`wizards`)
+Interactive text-based wizards that guide users through the setup and configuration of Partner Chains.
+
+**Usage:**
+```bash
+partner-chains-node wizards [SUBCOMMAND]
+```
+
+## Architecture
+
+### Command Structure
+
+The crate is built around the `PartnerChainsSubcommand` enum, which defines all available commands. Each command is implemented as a separate struct with its own configuration and execution logic.
+
+```rust
+pub enum PartnerChainsSubcommand<RuntimeBindings, PartnerchainAddress> {
+    SidechainParams(SidechainParamsCmd),
+    RegistrationStatus(RegistrationStatusCmd),
+    AriadneParameters(AriadneParametersCmd),
+    // ... other commands
+}
+```
+
+### Generic Design
+
+The crate is designed with generics to support different runtime configurations and address types:
+
+- `RuntimeBindings`: Specifies the Partner Chain runtime bindings
+- `PartnerchainAddress`: Defines the address format used by the specific Partner Chain
+
+### CLI Integration
+
+All commands implement the `CliConfiguration` trait from Substrate's CLI framework, ensuring consistent behavior and configuration management across the Partner Chains ecosystem.
+
+## Dependencies
+
+### Core Dependencies
+
+- **clap**: Command-line argument parsing
+- **sc-cli**: Substrate CLI framework integration
+- **sc-service**: Substrate service management
+- **sp-api**: Substrate runtime API definitions
+- **sp-runtime**: Substrate runtime primitives
+
+### Partner Chains Specific
+
+- **authority-selection-inherents**: Authority selection logic
+- **cli-commands**: Core CLI command implementations
+- **partner-chains-cli**: Partner Chains specific CLI utilities
+- **partner-chains-smart-contracts-commands**: Smart contract interaction commands
+- **sidechain-domain**: Sidechain domain types and primitives
+
+### Cryptography and Encoding
+
+- **parity-scale-codec**: SCALE codec for data serialization
+- **sp-core**: Substrate cryptographic primitives
+
+## Error Handling
+
+The crate uses Substrate's error handling patterns:
+
+- `sc_cli::Result<()>` for CLI operations
+- `sc_service::Error` for service-related errors
+- Custom error types for domain-specific operations
+
+Errors are automatically converted and propagated through the CLI framework, providing clear error messages to users.
+
+## Logging
+
+The crate includes sophisticated logging configuration:
+
+- **stderr**: Standard error output for general logging
+- **ogmios_client.log**: Dedicated file logging for Ogmios interactions
+- Configurable log levels for different components
+
+## Development
```

Comment:
We con't need this either. Besides that one could also mention testing docs running fmt and many other things. We dozens of crates here and I don't need repeating `cargo build -p <crate-name>` that many times in every readme.